app/src/main/java/com/onewelcome/showcaseapp/ui/screens/sections/SdkInitializationScreen.kt

```diff
@@ -2,52 +2,154 @@ package com.onewelcome.showcaseapp.ui.screens.sections
+    topBar = { TopBar(onNavigateBack) },
+  ) { innerPadding ->
     Column(
-      verticalArrangement = Arrangement.Center,
-      horizontalAlignment = Alignment.CenterHorizontally
+      modifier = Modifier
+        .padding(innerPadding)
+        .padding(start = Dimensions.standardPadding, end = Dimensions.standardPadding)
     ) {
+      SettingsSection(
+        modifier =
+          Modifier
+            .weight(1f)
+            .padding(bottom = Dimensions.smallPadding),
+        uiState = uiState,
+        onEvent = onEvent
+      )
       Button(
-        modifier = Modifier.wrapContentSize(),
-        onClick = { onInitializeSdkClicked.invoke() }
+        modifier = Modifier.fillMaxWidth(),
+        onClick = { onEvent(UiEvent.InitializeOneginiSdk) }
       ) {
         Text(stringResource(R.string.button_initialize_sdk))
       }
     }
   }
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun TopBar(onNavigateBack: () -> Unit) {
+  TopAppBar(
+    windowInsets = WindowInsets(0.dp),
+    title = { Text(stringResource(R.string.section_title_sdk_initialization)) },
+    navigationIcon = {
+      IconButton(onClick = onNavigateBack) {
+        Icon(
+          imageVector = Icons.AutoMirrored.Filled.ArrowBack,
+          contentDescription = stringResource(R.string.content_description_navigate_back)
+        )
+      }
+    })
+}
+
+@Composable
+private fun SettingsSection(modifier: Modifier, uiState: State, onEvent: (UiEvent) -> Unit) {
+  Column(
+    modifier = modifier
+      .verticalScroll(rememberScrollState()),
+    verticalArrangement = Arrangement.spacedBy(Dimensions.verticalSpacing)
+  ) {
+    Text(
+      text = stringResource(R.string.required),
+      style = MaterialTheme.typography.titleSmall
+    )
+    ExpandableCard(title = stringResource(R.string.label_title_handlers)) {} //TODO To be done in scope of EXAMPLEAND-153
+    Text(
+      text = stringResource(R.string.optional),
+      style = MaterialTheme.typography.titleSmall
+    )
+    ExpandableCard(
+      title = stringResource(R.string.label_http_settings)
+    ) { HttpSettings(uiState, onEvent) }
+    ExpandableCard(
+      title = stringResource(R.string.label_custom_authenticators)
+    ) { } //TODO To be done in scope of EXAMPLEAND-155
+    ExpandableCard(
+      title = stringResource(R.string.label_custom_identity_providers)
+    ) { } //TODO To be done in scope of EXAMPLEAND-156
+  }
+}
+
+@Composable
+private fun HttpSettings(uiState: State, onEvent: (UiEvent) -> Unit) {
+  Column(
+    modifier = Modifier.padding(Dimensions.standardPadding),
+    verticalArrangement = Arrangement.spacedBy(Dimensions.verticalSpacing)
+  ) {
+    SettingCheckbox(
+      text = stringResource(R.string.option_should_store_cookies),
+      checked = uiState.shouldStoreCookies
+    ) { onEvent(UiEvent.ChangeShouldStoreCookiesValue(it)) }
+    NumberSettingTextField(
+      modifier = Modifier.fillMaxWidth(),
+      value = uiState.httpConnectTimeout,
```

Comment:
Maybe we could add an `i` iconButton with tooltip? https://m3.material.io/components/tooltips/overview
We could add info on default values, as its not clear right now.